gossiper/gossiper.go

```diff
@@ -0,0 +1,21 @@
+package gossiper
+
+import "fmt"
+
+type gossiper struct {
+	name	string
+	ip		string
+	peers	map[string]gossiper
```

Comment:
Might want a flat map? IMO, we do not need the entire network information. Wdyt?

Reply:
Yeah we don't need the entire network information, I was thinking each node uses this map to keep its peers